packages/serverless-orchestration/test/ServerlessSpoke.js

```diff
@@ -186,26 +186,6 @@ describe("ServerlessSpoke.js", function () {
     ); // Check the process logger contained the error.
     assert.isTrue(lastSpyLogIncludes(spy, "Process exited with error")); // Check the process logger contains exit error.
   });
-  it("Serverless Spoke can correctly returns errors over http calls(invalid network identifier)", async function () {
```

Comment:
As a result of this change, the getWeb3() call became more permissive. The network name always falls back to localhost. It's no longer required to be in some standard list of allowed networks (since we don't want to rely on the truffle config anymore).